troubleshoot-data-inconsistency-errors.md

```diff
@@ -74,23 +74,21 @@ This error indicates that `index-values` are null and `record-values` are not nu
 
-- The data and indexes in the existing data are consistent and the current version of TiDB has a bug. If an ongoing transaction is about to write inconsistent data, TiDB aborts the transaction.
-- The data and indexes in the existing data are inconsistent. The inconsistent data could be from a dangerous operation performed by mistake in the past or caused by a TiDB bug.
-- The data and indexes are consistent but the detection algorithm has a bug that causes errors by mistake.
-
-If you receive a data inconsistency error, contact PingCAP technical support for troubleshooting immediately instead of dealing with the error by yourself. If PingCAP technical support confirms that the error is reported by mistake, or your application needs to skip such errors urgently, you can use the following methods to bypass the check.
+### Rewrite SQL
 
-### Disable error check
+If the data inconsistency error occurs in a particular SQL statement only, you can try bypassing the error by rewriting the SQL statement to another equivalent form using different execution operators.
 
-For the following errors reported in transaction execution, you can bypass the corresponding check:
+### Disable error checks
 
-- To bypass the check of errors 8138, 8139, and 8140, configure `set @@tidb_enable_mutation_checker=0`.
-- To bypass the check of error 8141, configure `set @@tidb_txn_assertion_level=OFF`.
+For the following errors reported in transaction execution, you can bypass the corresponding checks:
 
-For other errors reported in transaction execution and all errors reported during the execution of the `ADMIN CHECK [TABLE|INDEX]` statement, you cannot bypass the corresponding check, because the data inconsistency has already occurred.
+- To bypass the checks of errors 8138, 8139, and 8140, configure `set @@tidb_enable_mutation_checker=0`.
+- To bypass the checks of error 8141, configure `set @@tidb_txn_assertion_level=OFF`.
 
-### Rewrite SQL
+> **Note:**
+>
+> Disabling `tidb_enable_mutation_checker` and `tidb_txn_assertion_level` will bypass the corresponding checks of all SQL statements.
 
-Disabling `tidb_enable_mutation_checker` and `tidb_txn_assertion_level` mentioned in the previous section bypasses the corresponding check of all SQL statements. If an inconsistency error is misreported for a particular SQL statement, you can try bypassing the error by rewriting the SQL statement to another equivalent form using different execution operators.
\ No newline at end of file
+For other errors reported in transaction execution and all errors reported during the execution of the `ADMIN CHECK [TABLE|INDEX]` statement, you cannot bypass the corresponding check, because the data inconsistency has already occurred.
```

Comment:
```suggestion
For other errors reported in transaction execution and all errors reported during the execution of the `ADMIN CHECK [TABLE|INDEX]` statement, you cannot bypass the corresponding check, because the data is already inconsistent.
```